common/src/lib.rs

```diff
@@ -0,0 +1,22 @@
+// configure default clippy lints
+#![deny(clippy::correctness)]
+#![warn(clippy::complexity, clippy::style, clippy::perf, clippy::pedantic)]
+// disable some pedantic lints
+#![allow(
+    clippy::cast_possible_truncation,
+    clippy::cast_possible_wrap,
+    clippy::cast_precision_loss,
+    clippy::cast_sign_loss,
+    clippy::default_trait_access,
+    clippy::missing_errors_doc,
+    clippy::module_name_repetitions,
+    clippy::must_use_candidate,
+    clippy::non_ascii_literal,
+    clippy::similar_names,
+    clippy::single_match_else,
+    clippy::type_repetition_in_bounds,
+    clippy::used_underscore_binding, // TODO: remove wrt. https://github.com/rust-lang/rust-clippy/issues/5360
```

Comment:
This should not be used in `common` since it was only a workaround for `services`.

Reply:
The issue is fixed by the way, so it can also be removed in `services`.

Reply:
9b363240c265960a40eb6c94bcb3642e3007c538